### PR TITLE
Remove `Store.bind`

### PR DESCRIPTION
**Pull request by leodasvacas**
Essentially a revert of #60. Store is back to being just a namespace. Example is updated for our current preferred style, and the `dist` directory is added.

Also fixed small things that were off in `index.d.ts`.

Companion PR graphprotocol/graph-node#258

### CODE REVIEW

File: examples/example-event-handler/dist/ExampleSubgraph/abis/ExampleContract.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "anonymous": false,
+    "inputs": [{ "indexed": true, "name": "exampleParam", "type": "string" }],
+    "name": "ExampleEvent",
+    "type": "event"
+  }
+]
```
Comment (fordN) [inline examples/example-event-handler/dist/ExampleSubgraph/abis/ExampleContract.json]: I don't think we want to include the ./dist/ folder in the repo.  Might be best to add ./dist to .gitignore.

Comment (leodasvacas) [reply examples/example-event-handler/dist/ExampleSubgraph/abis/ExampleContract.json]: I think it's good, so when reviewing we can see the impact of a change on the example.

Comment (Jannis) [reply examples/example-event-handler/dist/ExampleSubgraph/abis/ExampleContract.json]: I can live with either approach.

File: index.d.ts [hunk 2]
```diff
@@ -215,7 +207,7 @@ declare class EthereumEvent {
   address: Address
   eventSignature: string
   blockHash: H256
-  params: Array<EthereumEventParam>
+  parameters: Array<EthereumEventParam>
```
Comment (Jannis) [inline index.d.ts]: I'm ok with renaming this field but it seems unnecessary.

Comment (leodasvacas) [reply index.d.ts]: The purpose is simply to match what's on `index.ts`

**Pull request merged**